docs/sources/helm-charts/mimir-distributed/release-notes/v4.1.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Grafana Mimir Helm chart version 4.1 release notes"
+menuTitle: "V4.1 release notes"
+description: "Release notes for Grafana Mimir Helm chart version 4.1"
+weight: 300
+---
+
+# Grafana Mimir Helm chart version 4.1 release notes
+
+Grafana Labs is excited to announce version 4.1 of the Grafana Mimir Helm chart, which is compatible with Grafana Mimir v2.6 and Grafana Enterprise Metrics v2.5. The `mimir-distributed` Helm chart is the best way to install Mimir on Kubernetes.
+
+To learn about new features and enhancements in Grafana Mimir or GEM, see the release notes for [Grafana Mimir v2.6](/docs/mimir/v2.6.x/release-notes/v2.6/) or the last release of [Grafana Enterprise Metrics v2.5](/docs/enterprise-metrics/latest/release-notes/v2-5/).
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For a comprehensive list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
+
+> **Note:** If you are upgrading from a previous major version, review the changelog entry for all the major releases since then and follow the necessary migrations.
```

Comment:
```suggestion
> **Note:** If you are upgrading from a previous major version, review the changelog entry for all the major releases since then and perform the necessary migrations.
```
Do we want them to perform the migrations, rather than just know about them?